src/cube_blog.js

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from 'react';
+import ReactDOM from 'react-dom';
+
+import BlogPost from './components/BlogPost';
+import Paginate from './components/Paginate';
+
+class CubeBlog extends Component {
+  constructor(props) {
+    super(props);
+  }
+
+  componentDidMount() {
+    updateBlog();
```

Comment:
This is a pretty fragile way to do a trigger and will lead to weird issues when the component updates - can you move the functions into event handlers in `BlogPost` instead?

Reply:
also could `blog.js` be a module that exports `updateBlog` rather than being on global namespace? to keep `updateBlog` from being a mystery function

Reply:
Yeah my vision would be to get rid of updateBlog altogether and have all the DOM manipulation + setting of event handlers done by react

Reply:
gotcha, +1

Reply:
Yes, I'm slowly moving blog.js into react as well